src/main/webapp/src/index.html

```diff
@@ -10,6 +10,7 @@
   <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
   <link rel="preconnect" href="https://fonts.gstatic.com">
   <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500&display=swap" rel="stylesheet">
+  <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyBLhQYnxnQldJaGpbA5athP8tTjMHo1YPk"></script>
```

Comment:
It would be preferable if the API Key is not pushed on Github. Have a look at these 2 PR (and maybe pull the code to use the API Key? Or wait until it merges to master if you prefer):
- API Key setup: https://github.com/googleinterns/step273-2020/pull/39
- How I use the config file to call the API key: https://github.com/googleinterns/step273-2020/pull/40
In the meanwhile, maybe remove the API key from the branch? And, for testing, you can add it again and remove it every time you push your code (it's a bit annoying, but it will only need to be done until you can use the API key config file)